frontend/src/components/FileUploader.js

```diff
@@ -78,6 +78,30 @@ Dropzone.propTypes = {
   id: PropTypes.string.isRequired,
 };
 
+export const getStatus = (status) => {
+  switch (status) {
+    case 'UPLOADING':
```

Comment:
Should we put these statuses in a constant somewhere?

Reply:
Yeah, I was wondering about that. certainly could, I did on the backend.

Reply:
I'm fine with leaving these here if it's the only place they're referenced. This function basically becomes the constant.